web/templates/set_status_overlay.hbs

```diff
@@ -9,17 +9,21 @@
         <i class="zulip-icon zulip-icon-close" aria-hidden="true"></i>
     </button>
 </div>
-<div class="user-status-options">
-    {{#each default_status_messages_and_emoji_info}}
-        <button type="button" class="button no-style user-status-value">
-            {{#if emoji.emoji_alt_code}}
-                <div class="emoji_alt_code">&nbsp;:{{emoji.emoji_name}}:</div>
-            {{else if emoji.url}}
-                <img src="{{emoji.url}}" class="emoji status-emoji" />
-            {{else}}
-                <div class="emoji status-emoji emoji-{{emoji.emoji_code}}"></div>
-            {{/if}}
-            {{status_text}}
-        </button>
-    {{/each}}
+<div class="user-status-options popover-menu">
```

Comment:
Abstractions wise, I would expect the `popover-menu` class to be on an outer div that is the entire popover, not have a `div` within the popover outside it. 

I bet this causes some sort of bug, but don't know precisely what.

@sayamsamal can you help get this cleaned up?

Reply:
Sure thing, added to TODO list for today!

Reply:
Yup, the CSS related to the `popover-menu` class is written in a way where it is expected that it would be the main parent div containing the `simplebar` divs inside it. So, things like the `border-radius` on it, and the `max-width` on the `simplebar-content` are some of the places which could negatively affect the modal.